livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaTemporalInterpreterTest.java

```diff
@@ -104,8 +104,8 @@ public void testInvalidAssertions() {
     ALATemporalInterpreter.interpretTemporal(er, tr);
 
     assertArrayEquals(
-        new String[] {OccurrenceIssue.RECORDED_DATE_UNLIKELY.name()},
-        tr.getIssues().getIssueList().toArray());
+        tr.getIssues().getIssueList().toArray(),
```

Comment:
Expected and Actual are the wrong way round.
See Error prone logs

Reply:
Checked others and pushed